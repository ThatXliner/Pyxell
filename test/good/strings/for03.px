
for t in "abcdefg" step -2, "ABC", "123456789" step 2 do
    x, y, z = t
    print "" + z + y + x
    if y >= 'C' do
        break
