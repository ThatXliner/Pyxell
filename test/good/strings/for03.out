1Ag
3Be
5Cc
